shell/common/engine.h

```diff
@@ -354,6 +354,19 @@ class Engine final : public RuntimeDelegate,
          fml::WeakPtr<SnapshotDelegate> snapshot_delegate,
          std::shared_ptr<VolatilePathTracker> volatile_path_tracker);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Create a Engine that shares as many resources as
+  ///             possible with the calling Engine such that together
+  ///             they occupy less memory.
```

Comment:
Same comment for doc. Can use more details.

Reply:
Done, mostly linked to other documentation.